corehq/apps/app_manager/suite_xml/post_process/instances.py

```diff
@@ -116,6 +116,7 @@ def __call__(self, fn):
 INSTANCE_BY_ID = {
     'groups': Instance(id='groups', src='jr://fixture/user-groups'),
     'reports': Instance(id='reports', src='jr://fixture/commcare:reports'),
+    'locations': Instance(id='locations', src='jr://fixture/commtrack:locations'),
```

Comment:
Updated to respect the feature flag, so `instances('locations')` will reference the flat fixture if it's available, but there's no way to reference both the flat and hierarchical fixture. If the flat is eventually going to replace the hierarchical, I think that'll be fine.